packages/notifier/src/notifier.js

```diff
@@ -220,7 +220,10 @@ export const makeNotifierFromAsyncIterable = asyncIterable => {
   });
 
   return Far('notifier', {
-    ...asyncIterable,
+    // Don't leak the original asyncIterableP since it may be remote and we also
+    // want the same semantics for this exposed iterable and the baseNotifier.
+    ...makeAsyncIterableFromNotifier(baseNotifier),
+    // TODO stop exposing baseNotifier methods directly
```

Comment:
This TODO will now appear in three places. It is appropriate in all three, or in none. I think it no longer makes sense, and so should be removed from all three rather than propagated.